src/main/components/DataTree.jsx

```diff
@@ -16,14 +17,20 @@ const DataTree = (props) => {
 
   useEffect(() => {
     const test = document.getElementById(`tree-${treeCount}`);
```

Comment:
since we are not using this any more, does this make sense here ? can we remove this completely ?